File: Projeto_5/src/FuncionarioAdm.py
from Usuario import Usuario

class FuncionarioAdm(Usuario):
    def __init__(self, nome, email, departamento, cargo):
        super().__init__(nome, email, "Funcionário Administrativo")
        self.departamento = departamento
        self.cargo = cargo

    def gerar_relatorio(self):
        relatorio = f"Funcionário Administrativo: {self.nome}\nDepartamento: {self.departamento}\nCargo: {self.cargo}"
        print(relatorio)

File: Projeto_5/src/FuncionarioAdm.cpp
#include "FuncionarioAdm.h"
#include <iostream>

using namespace std;

FuncionarioAdm::FuncionarioAdm(){}

FuncionarioAdm::FuncionarioAdm(
    string nome, string email, string departamento, string cargo)
    : Usuario(nome, email, "Funcionario"){
        this->departamento = departamento;
        this->cargo = cargo;
    }

    void FuncionarioAdm::gerarRelatorio() const{
        cout << "Funcionario" << nome << endl;
        cout << "Departamento " << departamento << endl;
        cout << "Cargo: " << cargo << endl;

    }